user_docs/en/changes.t2t

```diff
@@ -16,10 +16,13 @@ What's New in NVDA
 
 == Bug Fixes ==
 - The NVDA installer can now run from directories with special characters. (#31270)

```

Comment:
Consider fixing the issue number here while at it.

Reply:
Oops. good catch

```suggestion
- The NVDA installer can now run from directories with special characters. (#13270)
```